universe/user/user.go

```diff
@@ -49,12 +51,14 @@ type User struct {
 	bufferSends                 atomic.Bool
 	numSendsQueued              atomic.Int64
 	directLock                  sync.Mutex
+	offlineTimer                *generic.TimerSet[umid.UMID]
 }
 
 func NewUser(id umid.UMID, db database.DB) *User {
 	return &User{
-		id: id,
-		db: db,
+		id:           id,
+		offlineTimer: generic.NewTimerSet[umid.UMID](),
```

Comment:
Do we need to instantiate it for all users? Maybe leave it nil by default and instantiate and set for guest?